landoapi/spec/swagger.yml

```diff
@@ -52,7 +52,48 @@ paths:
           schema:
             allOf:
               - $ref: '#/definitions/Error'
+  /revisions/{revision_id}/transplants:
+    post:
+      operationId: landoapi.api.revisions.land
+      description: |
+        Sends request to transplant service and responds with id received
+      parameters:
+        - name: revision_id
+          in: path
+          type: string
+          description: |
+            The id of the revision to land. This can either be in form of
+            'D12345' (as seen in Phabricator URLs), or just the id number.
+          required: true
+        - name: api_key
+          in: query
+          type: string
+          description: |
+            A Phabricator Conduit API key to use to get the revision.
```

Comment:
yes, if it's the one defined in `docker-compose.yml` as `PHABRICATOR_UNPRIVILEGED_API_KEY=api-123456789`